mesonbuild/mlog.py

```diff
@@ -19,8 +19,23 @@
 information about Meson runs. Some output goes to screen,
 some to logging dir and some goes to both."""
 
+def _windows_ansi ():
+    ENABLE_VIRTUAL_TERMINAL_PROCESSING = 0x4
```

Comment:
__[Flake8]__

[F841] local variable 'ENABLE_VIRTUAL_TERMINAL_PROCESSING' is assigned to but never used


<!-- Comment by [SideCI](https://sideci.com) -->